packages/fast-usdc-contract/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -38,6 +38,13 @@ Generated by [AVA](https://avajs.dev).
         brandDenom: {
           'Alleged: USDC brand': 'ibc/FE98AAD68F02F03565E9FA39A5E627946699B2B07115889ED812D8BA639576A9',
         },
+        chainIdToChainName: {
```

Comment:
consider squashing the snapshot changes into the commit that breaks the snapshot